ext/native-decls/GetAllPeds.md

```diff
@@ -16,3 +16,22 @@ The data returned adheres to the following layout:
 
 ## Return value
 An object containing a list of peds handles.
+
+## Examples
+
+```lua
+--[[ 
+  Example =>
+     Using GetAllPeds, This example prints information of all peds, Ped id, Owner, Model and Armour.
+]]
+
+local allPeds = GetAllPeds() -- Get all peds handles known to the server.
+for i, ped in ipairs(allPeds) do
+    local pedOwner = NetworkGetEntityOwner(ped)
+    if pedOwner ~= nil and pedOwner then
```

Comment:
pedOwner can't be nil so this is redundant, if the entity has no owner it will be 0 iirc